_posts/2017-12-05-book-review-docker-up-running.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
```

Comment:
I do not like the overall layout/structure of the post.

May be it is the way paragraphs are posed, but the flow of the post doesn't feel soft. It jumps between things very fast.

Reply:
It might be hard but do you think https://github.com/ultimatecoder/Blog/pull/23/commits/7e6f4bc47378a916874d69123eca5e5717571c24 has mistakes but smooth flow? Thanks :)